source/core/slang-render-api-util.cpp

```diff
@@ -265,6 +266,11 @@ static bool _canLoadSharedLibrary(const char* libName)
     {
 #if SLANG_WINDOWS_FAMILY
         case RenderApiType::Vulkan: return _canLoadSharedLibrary("vulkan-1") || _canLoadSharedLibrary("vk_swiftshader");
+        case RenderApiType::WebGPU:
+            return
+                _canLoadSharedLibrary("webgpu_dawn") &&
+                _canLoadSharedLibrary("dxcompiler") &&
```

Comment:
This isn't addressed.

Reply:
Dawn compiles the WGSL we give it into something that it can use to render/compute. On Windows by default that is Directx12. In my WebGPU test application, Dawn wouldn't compile pipelines unless it could find dxcompiler.